### DISCUSSION TITLE
Connect several servo motors to an ESP32 (C3-M1)

### DISCUSSION THREAD

**Discussion by Cheonsa51** [Dlloydev/ESP32-ESP32S2-AnalogWrite#28] [General]
Hello, I have a beginner level in programming, I have to be able to move 6 servomotors,
3 sg90 9g and 2 SR402 P servo and the last one is hs422 servo on ESP32C3 M1

By following tutorials on the internet I was able to move a servomotor, I was also able to test them one by one, they are functional.
But when I tried to change the pin on the example, no more reaction nothing moved.

If anyone has an idea to help me

**Comment by Dlloydev** [comment]
Here's an [example](https://wokwi.com/projects/355852275661848577) on Wokwi.
